lib/bitcoin/versions/latest/NormalizedFeeCalculator.ts

```diff
@@ -72,6 +74,9 @@ export default class NormalizedFeeCalculator implements IFeeCalculator {
     // TODO: #943 This is left here because it may be versioned. Move it out if we confirm it will not be versioned.
     // https://github.com/decentralized-identity/sidetree/issues/943
     const blockMetadata = await this.blockMetadataStore.get(block, block + 1);
+    if (blockMetadata.length === 0) {
+      throw new SidetreeError(ErrorCode.NormalizedFeeCalculatorBlockNotFound);
```

Comment:
Please add reference to `NormalizedFeeCalculatorBlockNotFound` as an expected error thrown in the `IFeeCalculator.getNormalizedFee()` method because this now become part of the contract between lock monitor and versioned code.